src/client/pps.go

```diff
@@ -175,6 +174,18 @@ func (c APIClient) InspectJob(jobID string, blockState bool) (*pps.JobInfo, erro
 	return jobInfo, grpcutil.ScrubGRPC(err)
 }
 
+// InspectJobOutputCommit returns info about a job that created a commit.
+// blockState will cause the call to block until the job reaches a terminal state (failure or success).
+func (c APIClient) InspectJobOutputCommit(repoName, commitID string, blockState bool) (*pps.JobInfo, error) {
```

Comment:
It might make sense to eventually move the blockState type here to JobState similar to the way the commit interfacing functions use CommitState.